sycl/doc/design/DeviceIf.md

```diff
@@ -181,6 +197,7 @@ one of the following corresponding C++ macro names:
 * `__SYCL_TARGET_AMD_GPU_GFX906__`
 * `__SYCL_TARGET_AMD_GPU_GFX908__`
 * `__SYCL_TARGET_AMD_GPU_GFX90A__`
+* `__SYCL_TARGET_AMD_GPU_GFX90C__`
```

Comment:
Should we add macros for the other new targets here too?

Reply:
Yes. Thanks.